resources/markdown/who_audit_l2.md

```diff
@@ -0,0 +1,282 @@
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Continue to display (WHO) AUDIT-C screening instructions
+- Continue to display standard drink size graphic
+- Display Question #2  and Question #3 of the (WHO) AUDIT-C (Question #3 = How often do you have five or more drinks on one occasion?)
+- Record and display responses to Question #2 and Question #3
+- Calculate (WHO) AUDIT-C score
+- Record and display (WHO) AUDIT-C score
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 4: Consider presenting Question #9 and Question #10 of the (WHO) AUDIT when not administering the full (WHO) AUDIT: 
+
+## LOGIC PATH #1: Jump from Question #1 to Question #9 & Question #10 OR jump from end of (WHO) AUDIT-C to Question #9 & Question #10:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question response is 'YES'
+AND
+     Most recent (WHO) AUDIT-C Question #1 response is 'Never'
+OR
+     Most recent (WHO) AUDIT-C Question #1 score < 3
+     AND most recent (WHO) AUDIT-C Question #2 score = 0
+     AND most recent (WHO) AUDIT-C Question #3 score = 0
+     AND
+           Female
+          OR Sex at birth is unknown
+          Male > 65 years old 
+ OR   
+      Most recent (WHO) AUDIT-C Question #1 score < 4
+      AND most recent (WHO) AUDIT-C Question #2 score = 0
+      AND most recent (WHO) AUDIT-C Question #3 score = 0
+      AND Male
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Display Question #9 & Question #10 of the (WHO) AUDIT
+- Record and display responses to Question #9 & Question #10  of the (WHO) AUDIT
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 5: Consider additional screening: 
+
+## LOGIC PATH #1: Provide full (WHO) AUDIT:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+AND most recent Alcohol PS Question Response is 'YES'
+AND NOT most recent (WHO) AUDIT-C Q1 response is 'Never'
+AND most recent (WHO) AUDIT-C score >=3
+     AND Female
+     OR Male > 65 years old
+     OR Sex at birth is unknown
+OR most recent (WHO) AUDIT-C score >=4 AND Male <=65 years old
+</pre>
+
+### Exclusion:
+<pre>
+Carry over exclusions from Step 1
+</pre>
+
+### Interventions and Actions:
+<pre>
+- Display Question #4-Question #10 of the (WHO) AUDIT
+- Record and display responses to Question #4-Question #10 of the (WHO) AUDIT
+- Calculate score of (WHO) AUDIT
+- Record score of (WHO) AUDIT
+</pre>
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 6: Consider pregnancy status: 
+
+## LOGIC PATH #1: Post (WHO) AUDIT Questions #1, #9, and #10:
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old and < 50 years old
```

Comment:
SS: Although I could potentially combine Path #1 and Path #2, I'm concerned that doing so will make the logic harder to follow for someone not as familiar with the granular details in the guidance. As a result, I kept them separate.